commands/Transfering/transfer.js

```diff
@@ -36,8 +36,31 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(value) {
```

Comment:
please rename value to removalPoints

removalPoints - is negative. It is more pretty name I think